### ISSUE TITLE
Popup  is not in correct position

### ISSUE THREAD

**Issue by naveenkumarmallappa** [techery/progresshint#8]
I tried to give some constant height to layout but popup will raise in reverse order. 
And also these seekbars will not working in inside ScrollView and ListView.

Here I pasting my xml code with constant height:

```
<RelativeLayout
        android:layout_width="match_parent"
        android:layout_height="300dp">

        <android.support.percent.PercentFrameLayout
            android:layout_width="match_parent"
            android:layout_height="300dp"
            android:layout_marginBottom="40dp">
            

            <com.h6ah4i.android.widget.verticalseekbar.VerticalSeekBarWrapper
                android:layout_width="wrap_content"
                android:layout_height="300dp"
                android:layout_centerInParent="true"
                android:layout_marginTop="-10dp"
                android:layout_gravity="center">

                <io.techery.progresshint.addition.widget.VerticalSeekBar
                    android:id="@+id/seekbar_vertical"
                    android:layout_width="0dp"
                    android:layout_height="0dp"
                    android:progressDrawable="@drawable/seekbar_progressbar_default"
                    app:popupAlwaysShown="true"
                    app:popupLayout="@layout/custom_progress_hint_popup"
                    app:popupOffset="2dp"
                    app:popupStyle="follow"
                    app:seekBarRotation="CW270" />

            </com.h6ah4i.android.widget.verticalseekbar.VerticalSeekBarWrapper>

        </android.support.percent.PercentFrameLayout>

        </RelativeLayout>
```

**Comment by aarambhin** [comment]
Hi Navin,

I am facing same issue, have you found any solution for this?

**Comment by almozavr** [comment]
This is fixed with 0.2.4
